SublimeLinter.sublime-settings

```diff
@@ -54,29 +54,59 @@
     // What settings are available is documented in the readme of the
     // specific linter plugin.
     // Example:
-    // "linters": {
-    //     "somelinter": {
-    //         "disable": false,
-    //         "args": [],
-    //         "excludes": [],
-    //         "styles": [
-    //              {
-    //                  "icon": "diamond"
-    //              }
-    //         ]
-    //     },
-    //     "somepythonlinter": {
-    //         "python": null,
-    //         "disable": false,
-    //         "args": [
-    //             "--max-complexity", "-1",
-    //             "--max-line-length", "100",
-    //             "--ignore", "E731,E402"
-    //         ],
-    //         "excludes": [],
-    //     }
-    // }
-    "linters": {},
+    "linters": {
+        // The name of the linter you installed
+        "linter_name": {
```

Comment:
A fake setting doesn't do any harm. We don't evaluate this. And since we don't inherit these settings, as soon as the user defines even an empty 'linters' object in their own settings this fake 'linter_name' setting will be gone.

Of course, this is a detail, but I think it's easier to read, and of course it's parseable for automatic documentation.

Reply:
👍